doc/manuals/user/forbidden_characters.md

```diff
@@ -59,3 +59,42 @@ NGSIv2 introduces syntax restrictions for ID fields (such as entity id/type, att
 or metadata name/type) which are described in the "Field syntax restrictions" section in the
 [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/). You can also
 enable them for NGSIv1, as described in [this section of the documentation](v1_v2_coexistence.md#checking-id-fields).
+
+### Notification payload template
+
+NGSIv2 allows a templating mechanism (see "Notification templates" section in
+the [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/)). Forbidden
+characters restrictions apply to the `http.payload` field in NGSIv2 API operations, such as
+POST /v2/subscription or GET /v2/subscriptions.
+
+However, at notification time, any URL encoded characters is `http.payload` is decoded.
+
+Example:
+
+Let's consider the following `notification.http` object in a given subscription.
+
+```
+"http": {
+  "url": "http://foo.com/entity/${id}",
+  "headers": {
+    "Content-Type": "aplication/json"
+  },
+  "method": "PUT",
+  "qs": {
+    "type": "${type}"
+  },
+  "payload": "{ %22temperature%22: ${temperature}, %22asString%22: %22${temperature}%22 }"
```

Comment:
I think is better to be exact, I mean, to show in the example the request as it would be recieved by Orion "on the wire". So, we can do it somehow in the other way around, i.e. leave the example as it is now and include a note below like:

Note that the above `payload` value is the URL encoded version of this string: `{ "temperature": ${temperature}, "asString": "${temperature}" }`.